cpp/ql/src/semmle/code/cpp/Function.qll

```diff
@@ -979,19 +979,11 @@ class ConversionConstructor extends Constructor, ImplicitConversionFunction {
 }
 
 private predicate hasCopySignature(MemberFunction f) {
-  f.getParameter(0).getType()
-    .getUnderlyingType()                 // resolve typedefs
-    .(LValueReferenceType).getBaseType() // step through lvalue reference type
-    .getUnspecifiedType() =              // resolve typedefs, strip const/volatile
-  f.getDeclaringType()
+  f.getParameter(0).getUnspecifiedType().(LValueReferenceType).getBaseType() = f.getDeclaringType()
```

Comment:
I note that this change is deeper than just a change in documentation.  I agree that it's cleaner and should be correct.